schema/va-spec/base/experimental-functional-impact-study-statement-profile-source.yaml

```diff
@@ -0,0 +1,101 @@
+$schema: "https://json-schema.org/draft/2020-12/schema"
+$id: "https://w3id.org/ga4gh/schema/va-spec/1.x/base/experimental-functional-impact-study-statement-profile-source.yaml"  
+title: Experimental Variant Functional Impact Statement
+strict: true
+
+imports:
+  cat-vrs: ../../cat-vrs/cat-vrs-source.yaml
+  vrs: ../../vrs/vrs-source.yaml
+  gks-core: ../../gks-core/gks-core-source.yaml
+
+$defs:
+  ExperimentalVariantFunctionalImpactStudyStatement:
+    inherits: gks-core:Statement
+    maturity: draft
+    type: object
+    description: >-
+      A statement reporting a conclusion from a single assay or study about the functional
+      impact of a variant on a sequence feature (typically a gene or gene product).
+    properties:
+      type:
+        extends: type
+        const: "ExperimentalVariantFunctionalImpactStudyStatement"
+        default: "ExperimentalVariantFunctionalImpactStudyStatement"
+        description: MUST be "ExperimentalVariantFunctionalImpactStudyStatement".
+      subjectVariant:
+        extends: subject
+        oneOf:
+          - $ref: "/ga4gh/schema/vrs/2.x/json/MolecularVariation"
+          - $ref: "/ga4gh/schema/cat-vrs/1.x/json/CategoricalVariant"
+          - $ref: "/ga4gh/schema/gks-core/1.x/json/iriReference"
+        description: A protein or genomic contextual or canonical molecular variant.
+      predicate:  
+        extends: predicate
+        description: >-
+          The relationship this Statement describes between the subject Variant
+          and object Sequence Feature whose function it may alter.
+        const: impactsFunctionOf                
+      objectSequenceFeature:
+        extends: object
+        oneOf:
+          - $ref: "/ga4gh/schema/gks-core/1.x/json/iriReference"
+          - $ref: "/ga4gh/schema/gks-core/1.x/MappableConcept"      # See https://github.com/ga4gh/gks-common/pull/77
```

Comment:
@mbrush can we archive/remove the comment on the end of this line or do we need to still visit the proposal of creating a `SequenceFeature` class?

Reply:
I agree, let's drop the comment and notion of creating a SequenceFeature class for now. `MappableConcept` should cover us for our immediate use case.

Reply:
sticking with MappableConcept for now. Will revisit specialized domain entities as we evolve.